### ISSUE TITLE
PDO transmission type is not written to OD via setter

### ISSUE THREAD

**Issue by phettberg** [semiversus/python-durand#1]
I take the README example:
```python
node.tpdo[0].mapping = [(0x2001, 1), (0x2001, 2)]
node.tpdo[0].transmission_type = 1  # transmit on every SYNC

node.rpdo[0].mapping = [(0x2000, 0)]
node.tpdo[0].transmission_type = 255  # event driven (processed when received)
```
When setting `.transmission_type` to `255` it works, because `255` is the default value (I'm not completely sure how that works). However, when setting to any other value, the PDO sync scheduler is indeed configured, but the object dictionary for PDO communication parameters is not updated:
* 0x1800 + tpdo_index for TPDO
* 0x1400 + rpdo_index for RPDO

IMHO, the setter should write it to the OD, as the setter of `.inhibit_time` does it as well:
```python
    @inhibit_time.setter
    def inhibit_time(self, value: float):
        return self._node.object_dictionary.write(
            0x1800 + self._index, 3, value * 10_000
        )
```

For transmission type it should be added in tpdo and rpdo, at this time it's missing:
```python
    def _set_transmission_type(self, value: int):
        if self._sync_handler:
            self._sync_handler.cancel()
            self._sync_handler = None

        if value <= 240:
            self._sync_handler = SyncHandler(self, value)

        self._transmission_type = value
```

**Comment by semiversus** [comment]
You're right. Will check it to see if any side effects could happen. Thanks!

**Comment by semiversus** [comment]
Fixed in 0.4.12, here is the fix commit: e1aa1e7c8a33bd4f4902ae8245e3ace2a45f3107

**Comment by phettberg** [comment]
Thank you so much for the quick response! I just wanted to implement a workaround when I received your notification. :)
